config.w32

```diff
@@ -137,7 +137,8 @@ if (PHP_MONGODB != "no") {
     BSON_HAVE_TIMESPEC: 0,
     BSON_EXTRA_ALIGN: 0,
     BSON_HAVE_GMTIME_R: 0,
-    BSON_HAVE_RAND_R: 0
+    BSON_HAVE_RAND_R: 0,
+    BSON_HAVE_ALIGNED_ALLOC: 0
```

Comment:
@eramongodb: Please confirm that my assumption is correct about `aligned_alloc` being unsupported on Windows. I inferred this from the previous logic in https://github.com/mongodb/mongo-c-driver/commit/2a35499c2ac9e66da7cc6eb3223b459f63fba458#diff-1405be263deb1a28987c54e138f2d9162df037b9965a028a1e376bdd8bb2a996L34.